include/psp2/ctrl.h

```diff
@@ -82,14 +87,32 @@ typedef struct SceCtrlData {
 	unsigned char 	rx;
 	/** Right analogue stick, Y axis. */
 	unsigned char 	ry;
-	/** Reserved. */
-	uint8_t		reserved0[4];
+	/** Up button */
+	uint8_t		up;
+	/** Right button */
+	uint8_t		right;
+	/** Down button */
+	uint8_t		down;
+	/** Left button */
+	uint8_t		left;
 	/** Left trigger (L2) */
-	unsigned char	lt;
+	uint8_t		l2;
 	/** Right trigger (R2) */
-	unsigned char	rt;
+	uint8_t		r2;
```

Comment:
it can make compatibility issue.
can you remain lt and rt?

Reply:
I think it would be better to name it r2/l2 due to the fact we have also l1/t2 references.
Afaik, the only plugins using these are @xerpi ones to adapt DS3/DS4 to Vita via BT.

Reply:
personally, I like l2/r2 than lt/rt too. but the library should have low broken changes. (although that was bad decision)
these fields can use all homebrews not only use plugins. (for example moonlight used [these][1])

[1]: https://github.com/xyzz/vita-moonlight/blob/8c71034/src/input/vita.c#L454-L455